carts/views.py

```diff
@@ -44,3 +44,16 @@ def post(self, request):
         except KeyError:
             return JsonResponse({"message" : "KEY_ERROR"}, status=400)
 
+    @login_decorator
+    def patch(self, request, cart_id):
+            data           = json.loads(request.body)
+            quantity       = data['quantity']
+            
+            if quantity <= 0:
+                return JsonResponse({'message':'QUANTITY_ERROR'}, status=400)
+
+            cart = Cart.objects.get(id=cart_id)
```

Comment:
get method에 대한 예외처리 추가

```suggestion
            cart = Cart.objects.get(id=cart_id, user=request.user)
```